mohanadarafe/GuideMe
# Automated End to End Tests (#176)

The following end2end tests are now able to run with up to date master code.

The 4 test scenarios are as follows

TEST 1
The following source code represents the following automated test scenario
 1. Load the app
2. Display main screen
3. Verify the map is present
4. Click the toggle button to switch the map to Loyola
5. Verifies the map is present
6. Returns back to SGW campus


TEST 2
The following source code represents the following automated test scenario
1. Load the app
2. Display main screen
3. Click on Hall Building from the Map
4. Checks if the bottom menu changed
5. Click the GET INSIDE button
6. Change floors
 7. Scroll across the floor map
 8. Click EXIT BUILDING button


TEST 3
The following source code represents the following automated test scenario
1. Load the app
 2. Display main screen
3. Click on Hall Building from the Map
4. Checks if the bottom menu changed
5. Click on the bottom menu
6. Verify the moreInfoPage is present
7. Scroll through the different departments and faculties
8. Tap the arrow down key


TEST 4 
The following source code represents the following automated test scenario
1. Load the app
2. Display main screen
3. Click on the searchbar
4. Type the query: "hall building"
5. Select the corresponding item from the dropdown list
6. Click Get DIRECTIONS button
